openstf/feature_engineering/lag_features.py

```diff
@@ -86,10 +90,14 @@ def extract_lag_features(feature_names: List[str]) -> Tuple[list, list]:
         elif number_of_days is not None:
             days_list.append(int(number_of_days[1]))
 
+    # Discard lag times that are not available for the specified horizon
+    minutes_list = list(set([i for i in minutes_list if i > horizon * 60]))
```

Comment:
just checking, Is the sign the right direction here? Should it not be the other way around -> `if i <= horizon * 60`. Also why is `set` required, can we have duplicates here and why? Perhaps explain when applicable.